src/components/landing/Landing.module.scss

```
@@ -1,7 +1,7 @@
 @import "/styles/variables.scss";
 
 .landingContainer {
-  background-image: url("../../../public/1c0f146c96b2dc1b9ae15ff133378633.jpeg");
+  background-image: url("https://fd-internal-development.imgix.net/xn2YPAK6UgWU1-7YZ6fgvQkGmok.jpg");
```

```suggestion
  background-image: url("https://fd-internal-development.imgix.net/xn2YPAK6UgWU1-7YZ6fgvQkGmok.jpg?q=45");
```

Add at least an IMGIX query parameter to the URL so it's certain using a rendered version of the image and never the original image. Original images costs a lot of money to download.